terraform-common.mk

```diff
@@ -0,0 +1,41 @@
+ENV_NAME := $(notdir $(shell cd $(PWD) && pwd))
```

Comment:
what is the purpose of moving this from terraform.mk and doing `+include $(shell git rev-parse --show-toplevel)/terraform-common.mk` in original file?